src/parser/expression.js

```diff
@@ -316,7 +316,7 @@ pp.parseSubscripts = function (base, startPos, startLoc, noCalls) {
     } else if (this.match(tt.backQuote)) {
       let node = this.startNodeAt(startPos, startLoc);
       node.tag = base;
-      node.quasi = this.parseTemplate();
+      node.quasi = this.parseTemplate(true);
```

Comment:
Come to think, the same applies to using a default parameter for `readCodePoint`.